### ISSUE TITLE
Is onepager still being developed?

### ISSUE THREAD

**Issue by nathobson** [themexpert/onepager#123]
I've only just stumbled across onepager. Is it still being developed? I can see the most recent commit is 5 months ago and that issues aren't being replied to by the team. It was interesting to read the 2.0 milestones but it doesn't seem to have come to fruition.

**Comment by BurlesonBrad** [comment]
After revisiting this, and contemplating it for a client, it doesn't really look like it. _but I could definitely be wrong_ :stuck_out_tongue_winking_eye: 

If you'll click and look at the two main devs, then click on their public activity tab, you'll see that it hasn't been touched, merged, or committed on for quite some time.

To be fair, I think a tag to @codexpert would be respectable and prudent. :+1:

**Comment by BurlesonBrad** [comment]
https://github.com/themexpert/onepager/graphs/contributors

**Comment by codexpert** [comment]
Hey Guys,

To be honest, we didn't touch Onepager code because its stable version working just fine. We have had plan for few updates. We were busy with our latest state-of-the-art page builder Quix and its released for Joomla here https://www.themexpert.com/quix-pagebuilder

We'll take Quix to WordPress and other platform like Flat file CMS. We built Quix platform independent so we can port it any CMS we want :)

**Comment by byronyasgur** [comment]
@codexpert so is the plan to move to Quix on WordPress instead of onepager (ie to phase out onepager on WordPress) or do you plan on supporting both projects going forward.  BTW there are apparently js bugs with onepager on WP 4.5.1 - at least going by the latest chatter on the gitter board https://gitter.im/themexpert/onepager havent tested it myself

**Comment by DavidBruchmann** [comment]
regrettable news :-(
I waited already for a new version.
But perhaps it's better like this because how contents are stored in
database is terrible.

**Comment by nadimtuhin** [comment]
It was very hard for us to discontinue onepager. Our survey resulted on a backend version of pagebuilder. So we are working on Quix a backend page builder. We have plans to add a frontend builder on Quix. If that happens and gets critical reception we will port it in WordPress.

**Comment by byronyasgur** [comment]
@nadimtuhin is that your confirmation that onepager is being discontinued then? Your post is a bit confusing in a number of ways IMO and I know there's a bit of a community so I'm sure people would like an actually official announcement - a clear one.

**Comment by codexpert** [comment]
No, we are not discontinuing OnePage, In fact we releasing new WP themes based on OnePager every month. We have no plan to discontinue now

**Comment by byronyasgur** [comment]
wow - very confusing - @nadimtuhin sounded like he was saying you were - glad to hear otherwise

**Comment by codexpert** [comment]
Hello Guys,

We started the v2 of OnePager and this time it would be awesome
